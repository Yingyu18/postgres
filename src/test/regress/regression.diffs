diff -U3 /Users/yingyuliu/Desktop/postgres0530/src/test/regress/expected/create_index.out /Users/yingyuliu/Desktop/postgres0530/src/test/regress/results/create_index.out
--- /Users/yingyuliu/Desktop/postgres0530/src/test/regress/expected/create_index.out	2024-05-30 17:06:51.000000000 +0800
+++ /Users/yingyuliu/Desktop/postgres0530/src/test/regress/results/create_index.out	2024-05-31 12:00:19.000000000 +0800
@@ -1954,7 +1954,7 @@
 ORDER BY thousand;
  thousand | tenthous 
 ----------+----------
-        0 |     3000
+        2 |        2
         1 |     1001
 (2 rows)
 
diff -U3 /Users/yingyuliu/Desktop/postgres0530/src/test/regress/expected/vacuum_parallel.out /Users/yingyuliu/Desktop/postgres0530/src/test/regress/results/vacuum_parallel.out
--- /Users/yingyuliu/Desktop/postgres0530/src/test/regress/expected/vacuum_parallel.out	2024-05-30 17:06:51.000000000 +0800
+++ /Users/yingyuliu/Desktop/postgres0530/src/test/regress/results/vacuum_parallel.out	2024-05-31 12:00:31.000000000 +0800
@@ -34,7 +34,7 @@
   pg_size_bytes(current_setting('min_parallel_index_scan_size'));
  trigger_parallel_vacuum_nindexes 
 ----------------------------------
-                                2
+                                1
 (1 row)
 
 -- Parallel VACUUM with B-Tree page deletions, ambulkdelete calls:
diff -U3 /Users/yingyuliu/Desktop/postgres0530/src/test/regress/expected/domain.out /Users/yingyuliu/Desktop/postgres0530/src/test/regress/results/domain.out
--- /Users/yingyuliu/Desktop/postgres0530/src/test/regress/expected/domain.out	2024-05-30 17:06:51.000000000 +0800
+++ /Users/yingyuliu/Desktop/postgres0530/src/test/regress/results/domain.out	2024-05-31 12:00:34.000000000 +0800
@@ -1290,10 +1290,8 @@
   ORDER BY domain_name;
  domain_catalog | domain_schema | domain_name | table_catalog | table_schema | table_name | column_name 
 ----------------+---------------+-------------+---------------+--------------+------------+-------------
- regression     | public        | con         | regression    | public       | domcontest | col1
  regression     | public        | dom         | regression    | public       | domview    | col1
- regression     | public        | things      | regression    | public       | thethings  | stuff
-(3 rows)
+(1 row)
 
 SELECT * FROM information_schema.domain_constraints
   WHERE domain_name IN ('con', 'dom', 'pos_int', 'things')
@@ -1311,11 +1309,8 @@
   ORDER BY domain_name;
  domain_catalog | domain_schema | domain_name | data_type | character_maximum_length | character_octet_length | character_set_catalog | character_set_schema | character_set_name | collation_catalog | collation_schema | collation_name | numeric_precision | numeric_precision_radix | numeric_scale | datetime_precision | interval_type | interval_precision | domain_default | udt_catalog | udt_schema | udt_name | scope_catalog | scope_schema | scope_name | maximum_cardinality | dtd_identifier 
 ----------------+---------------+-------------+-----------+--------------------------+------------------------+-----------------------+----------------------+--------------------+-------------------+------------------+----------------+-------------------+-------------------------+---------------+--------------------+---------------+--------------------+----------------+-------------+------------+----------+---------------+--------------+------------+---------------------+----------------
- regression     | public        | con         | integer   |                          |                        |                       |                      |                    |                   |                  |                |                32 |                       2 |             0 |                    |               |                    |                | regression  | pg_catalog | int4     |               |              |            |                     | 1
  regression     | public        | dom         | integer   |                          |                        |                       |                      |                    |                   |                  |                |                32 |                       2 |             0 |                    |               |                    |                | regression  | pg_catalog | int4     |               |              |            |                     | 1
- regression     | public        | pos_int     | integer   |                          |                        |                       |                      |                    |                   |                  |                |                32 |                       2 |             0 |                    |               |                    |                | regression  | pg_catalog | int4     |               |              |            |                     | 1
- regression     | public        | things      | integer   |                          |                        |                       |                      |                    |                   |                  |                |                32 |                       2 |             0 |                    |               |                    |                | regression  | pg_catalog | int4     |               |              |            |                     | 1
-(4 rows)
+(1 row)
 
 SELECT * FROM information_schema.check_constraints
   WHERE (constraint_schema, constraint_name)
