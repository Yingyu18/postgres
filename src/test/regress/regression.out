# using postmaster on Unix socket, default port
ok 1         - test_setup                                400 ms
# parallel group (20 tests):  char varchar oid pg_lsn text int2 int4 money uuid float4 regproc boolean int8 float8 bit rangetypes txid numeric name enum
ok 2         + boolean                                   109 ms
ok 3         + char                                       54 ms
ok 4         + name                                      277 ms
ok 5         + varchar                                    57 ms
ok 6         + text                                       80 ms
ok 7         + int2                                       82 ms
ok 8         + int4                                       87 ms
ok 9         + int8                                      110 ms
ok 10        + oid                                        67 ms
ok 11        + float4                                     96 ms
ok 12        + float8                                    114 ms
ok 13        + bit                                       139 ms
ok 14        + numeric                                   271 ms
ok 15        + txid                                      268 ms
ok 16        + uuid                                       88 ms
ok 17        + enum                                      280 ms
ok 18        + money                                      78 ms
ok 19        + rangetypes                                258 ms
ok 20        + pg_lsn                                     59 ms
ok 21        + regproc                                    82 ms
# parallel group (20 tests):  md5 lseg path line circle time point timetz macaddr numerology macaddr8 inet date interval polygon box strings multirangetypes timestamp timestamptz
ok 22        + strings                                   170 ms
ok 23        + md5                                        25 ms
ok 24        + numerology                                 98 ms
ok 25        + point                                      67 ms
ok 26        + lseg                                       36 ms
ok 27        + line                                       39 ms
ok 28        + box                                       152 ms
ok 29        + path                                       37 ms
ok 30        + polygon                                   140 ms
ok 31        + circle                                     53 ms
ok 32        + date                                      113 ms
ok 33        + time                                       59 ms
ok 34        + timetz                                     75 ms
ok 35        + timestamp                                 372 ms
ok 36        + timestamptz                               389 ms
ok 37        + interval                                  128 ms
ok 38        + inet                                      105 ms
ok 39        + macaddr                                    75 ms
ok 40        + macaddr8                                   91 ms
ok 41        + multirangetypes                           254 ms
# parallel group (12 tests):  comments unicode misc_sanity xid expressions geometry horology regex type_sanity mvcc tstypes opr_sanity
ok 42        + geometry                                  221 ms
ok 43        + horology                                  253 ms
ok 44        + tstypes                                   408 ms
ok 45        + regex                                     297 ms
ok 46        + type_sanity                               336 ms
ok 47        + opr_sanity                               6203 ms
ok 48        + misc_sanity                                91 ms
ok 49        + comments                                   66 ms
ok 50        + expressions                               149 ms
ok 51        + unicode                                    85 ms
ok 52        + xid                                       135 ms
ok 53        + mvcc                                      379 ms
# parallel group (5 tests):  copyselect copydml copy insert_conflict insert
ok 54        + copy                                      117 ms
ok 55        + copyselect                                 38 ms
ok 56        + copydml                                    43 ms
ok 57        + insert                                    413 ms
ok 58        + insert_conflict                           252 ms
# parallel group (7 tests):  create_function_c create_operator create_procedure create_type create_schema create_misc create_table
ok 59        + create_function_c                          41 ms
ok 60        + create_misc                               138 ms
ok 61        + create_operator                            81 ms
ok 62        + create_procedure                          113 ms
ok 63        + create_table                              534 ms
ok 64        + create_type                               111 ms
ok 65        + create_schema                             116 ms
# parallel group (5 tests):  index_including_gist create_index_spgist index_including create_view create_index
not ok 66    + create_index                              722 ms
ok 67        + create_index_spgist                       231 ms
ok 68        + create_view                               345 ms
ok 69        + index_including                           300 ms
ok 70        + index_including_gist                      195 ms
# parallel group (16 tests):  create_cast hash_func errors roleattributes create_aggregate infinite_recurse select drop_if_exists typed_table create_function_sql create_am constraints vacuum updatable_views inherit triggers
ok 71        + create_aggregate                          128 ms
ok 72        + create_function_sql                       234 ms
ok 73        + create_cast                                75 ms
ok 74        + constraints                               353 ms
ok 75        + triggers                                 1386 ms
ok 76        + select                                    143 ms
ok 77        + inherit                                  1057 ms
ok 78        + typed_table                               212 ms
ok 79        + vacuum                                    465 ms
ok 80        + drop_if_exists                            151 ms
ok 81        + updatable_views                           874 ms
ok 82        + roleattributes                            102 ms
ok 83        + create_am                                 258 ms
ok 84        + hash_func                                  81 ms
ok 85        + errors                                     88 ms
ok 86        + infinite_recurse                          122 ms
ok 87        - sanity_check                              129 ms
# parallel group (20 tests):  delete select_distinct_on select_having select_implicit case namespace prepared_xacts select_into random portals transactions select_distinct arrays hash_index update btree_index union subselect join aggregates
ok 88        + select_into                               186 ms
ok 89        + select_distinct                           419 ms
ok 90        + select_distinct_on                        106 ms
ok 91        + select_implicit                           142 ms
ok 92        + select_having                             115 ms
ok 93        + subselect                                3211 ms
ok 94        + union                                    2612 ms
ok 95        + case                                      158 ms
ok 96        + join                                     3730 ms
ok 97        + aggregates                               4391 ms
ok 98        + transactions                              409 ms
ok 99        + random                                    250 ms
ok 100       + portals                                   330 ms
ok 101       + arrays                                    572 ms
ok 102       + btree_index                              1435 ms
ok 103       + hash_index                               1148 ms
ok 104       + update                                   1228 ms
ok 105       + delete                                     81 ms
ok 106       + namespace                                 165 ms
ok 107       + prepared_xacts                            168 ms
# parallel group (20 tests):  init_privs drop_operator security_label tablesample lock password collate groupingsets object_address replica_identity gin gist spgist identity matview generated rowsecurity brin join_hash privileges
ok 108       + brin                                     1336 ms
ok 109       + gin                                       823 ms
ok 110       + gist                                      829 ms
ok 111       + spgist                                    849 ms
ok 112       + privileges                               1604 ms
ok 113       + init_privs                                 26 ms
ok 114       + security_label                            160 ms
ok 115       + collate                                   483 ms
ok 116       + matview                                   894 ms
ok 117       + lock                                      226 ms
ok 118       + replica_identity                          642 ms
ok 119       + rowsecurity                              1127 ms
ok 120       + object_address                            538 ms
ok 121       + tablesample                               219 ms
ok 122       + groupingsets                              479 ms
ok 123       + drop_operator                             113 ms
ok 124       + password                                  302 ms
ok 125       + identity                                  854 ms
ok 126       + generated                                1051 ms
ok 127       + join_hash                                1330 ms
# parallel group (2 tests):  brin_bloom brin_multi
ok 128       + brin_bloom                                 86 ms
ok 129       + brin_multi                                168 ms
# parallel group (17 tests):  collate.icu.utf8 async dbsize collate.utf8 tidrangescan tidscan tid tsrf misc alter_operator sysviews create_role misc_functions incremental_sort alter_generic merge create_table_like
ok 130       + create_table_like                         450 ms
ok 131       + alter_generic                             230 ms
ok 132       + alter_operator                            145 ms
ok 133       + misc                                      128 ms
ok 134       + async                                      37 ms
ok 135       + dbsize                                     60 ms
ok 136       + merge                                     364 ms
ok 137       + misc_functions                            185 ms
ok 138       + sysviews                                  152 ms
ok 139       + tsrf                                      119 ms
ok 140       + tid                                       100 ms
ok 141       + tidscan                                    96 ms
ok 142       + tidrangescan                               94 ms
ok 143       + collate.utf8                               73 ms
ok 144       + collate.icu.utf8                           26 ms
ok 145       + incremental_sort                          181 ms
ok 146       + create_role                               166 ms
# parallel group (7 tests):  collate.linux.utf8 collate.windows.win1252 amutils psql_crosstab rules psql stats_ext
ok 147       + rules                                     335 ms
ok 148       + psql                                      428 ms
ok 149       + psql_crosstab                              42 ms
ok 150       + amutils                                    41 ms
ok 151       + stats_ext                                1188 ms
ok 152       + collate.linux.utf8                         14 ms
ok 153       + collate.windows.win1252                    14 ms
ok 154       - select_parallel                          1417 ms
ok 155       - write_parallel                             88 ms
not ok 156   - vacuum_parallel                            47 ms
# parallel group (2 tests):  subscription publication
ok 157       + publication                               498 ms
ok 158       + subscription                               45 ms
# parallel group (17 tests):  portals_p2 combocid advisory_lock xmlmap functional_deps equivclass guc tsdicts dependency select_views bitmapops indirect_toast tsearch window cluster foreign_data foreign_key
ok 159       + select_views                              244 ms
ok 160       + portals_p2                                117 ms
ok 161       + foreign_key                              1761 ms
ok 162       + cluster                                   551 ms
ok 163       + dependency                                241 ms
ok 164       + guc                                       232 ms
ok 165       + bitmapops                                 332 ms
ok 166       + combocid                                  149 ms
ok 167       + tsearch                                   402 ms
ok 168       + tsdicts                                   237 ms
ok 169       + foreign_data                              675 ms
ok 170       + window                                    414 ms
ok 171       + xmlmap                                    159 ms
ok 172       + functional_deps                           213 ms
ok 173       + advisory_lock                             145 ms
ok 174       + indirect_toast                            348 ms
ok 175       + equivclass                                212 ms
# parallel group (9 tests):  json_encoding jsonpath_encoding jsonpath sqljson sqljson_jsontable jsonb_jsonpath json sqljson_queryfuncs jsonb
ok 176       + json                                      119 ms
ok 177       + jsonb                                     177 ms
ok 178       + json_encoding                              33 ms
ok 179       + jsonpath                                   45 ms
ok 180       + jsonpath_encoding                          38 ms
ok 181       + jsonb_jsonpath                            107 ms
ok 182       + sqljson                                    75 ms
ok 183       + sqljson_queryfuncs                        152 ms
ok 184       + sqljson_jsontable                          98 ms
# parallel group (18 tests):  prepare returning conversion plancache xml temp rowtypes polymorphism sequence with limit copy2 largeobject rangefuncs domain truncate plpgsql alter_table
ok 185       + plancache                                 218 ms
ok 186       + limit                                     479 ms
ok 187       + plpgsql                                  1018 ms
ok 188       + copy2                                     497 ms
ok 189       + temp                                      320 ms
not ok 190   + domain                                    597 ms
ok 191       + rangefuncs                                524 ms
ok 192       + prepare                                    78 ms
ok 193       + conversion                                193 ms
ok 194       + truncate                                  612 ms
ok 195       + alter_table                              1914 ms
ok 196       + sequence                                  419 ms
ok 197       + polymorphism                              411 ms
ok 198       + rowtypes                                  388 ms
ok 199       + returning                                 156 ms
ok 200       + largeobject                               510 ms
ok 201       + with                                      471 ms
ok 202       + xml                                       256 ms
# parallel group (15 tests):  predicate hash_part reloptions explain partition_info compression partition_aggregate partition_merge memoize partition_split partition_prune partition_join stats tuplesort indexing
ok 203       + partition_merge                           922 ms
ok 204       + partition_split                          1428 ms
ok 205       + partition_join                           1636 ms
ok 206       + partition_prune                          1586 ms
ok 207       + reloptions                                153 ms
ok 208       + hash_part                                  93 ms
ok 209       + indexing                                 2192 ms
ok 210       + partition_aggregate                       893 ms
ok 211       + partition_info                            181 ms
ok 212       + tuplesort                                1665 ms
ok 213       + explain                                   154 ms
ok 214       + compression                               286 ms
ok 215       + memoize                                  1145 ms
ok 216       + stats                                    1631 ms
ok 217       + predicate                                  71 ms
# parallel group (2 tests):  oidjoins event_trigger
ok 218       + oidjoins                                 2778 ms
ok 219       + event_trigger                            2825 ms
ok 220       - event_trigger_login                        24 ms
ok 221       - fast_default                              268 ms
ok 222       - tablespace                                472 ms
1..222
# 3 of 222 tests failed.
# The differences that caused some tests to fail can be viewed in the file "/Users/yingyuliu/Desktop/postgres0530/src/test/regress/regression.diffs".
# A copy of the test summary that you see above is saved in the file "/Users/yingyuliu/Desktop/postgres0530/src/test/regress/regression.out".
